corsheaders/models.py

```diff
@@ -3,3 +3,6 @@
 
 class CorsModel(models.Model):
     cors = models.CharField(max_length=255)
+    
+    def __unicode__(self)
```

Comment:
Here's a guide with my thought process spelled out:

1. Find out where to put the test. There are two test files in the `tests` folder, `test_middleware` and `test_conf`. We want to test just the model, which is in `models` and not `middleware` or `conf`, so it looks like we need to create `test_models.py`.
2. Inside this file create a new test case for the model called `CorsModelTests`, inheriting from Django's `TestCase` (`from django.test import TestCase`).
3. The method we want to test is pretty simple. We just need to check that calling `unicode()` of a model instance is working, so we can just compare it to the value of `cors` of that instance. So we can write a test inside our test case:

```py
def test_str(self):
    instance = CorsModel(cors='foo')
    assert unicode(instance) == 'foo'
```

We will need to import the model at the top of the file with `from corsheaders.models import CorsModel`. Because we want to compare to a unicode string, we should add `from __future__ import unicode_literals` at the top of the file, rather than use the `u''` string format.
4. Run the tests with `tox`. Aha! This test breaks with Python 3:

```
_____________________________________________________________ CorsModelTests.test_str ______________________________________________________________

self = <tests.test_models.CorsModelTests testMethod=test_str>

    def test_str(self):
        instance = CorsModel(cors='foo')
>       assert unicode(instance) == 'foo'
E       NameError: name 'unicode' is not defined
```

Of course, `unicode` isn't a thing there - it was unified with `str`. We should really be using `six.text_type` rather than `unicode` to handle this mapping. Fix the test to add `from django.utils import six` and change the line to be `assert six.text_type(instance) == 'foo'`.

5. Run tests again with `tox` - they pass. Yay.

In the end I came up with this patch:

```
diff --git a/corsheaders/models.py b/corsheaders/models.py
index c4c774a..1f9097b 100644
--- a/corsheaders/models.py
+++ b/corsheaders/models.py
@@ -1,9 +1,10 @@
 from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
 
 
+@python_2_unicode_compatible
 class CorsModel(models.Model):
     cors = models.CharField(max_length=255)
 
-    @python2_unicode_compatible
-    def __str__(self)
+    def __str__(self):
         return self.cors
diff --git a/tests/test_models.py b/tests/test_models.py
new file mode 100644
index 0000000..d0c9e43
--- /dev/null
+++ b/tests/test_models.py
@@ -0,0 +1,11 @@
+from django.test import TestCase
+from django.utils import six
+
+from corsheaders.models import CorsModel
+
+
+class CorsModelTests(TestCase):
+
+    def test_str(self):
+        instance = CorsModel(cors='foo')
+        assert six.text_type(instance) == 'foo'

```